core/controllers/suggestion.py

```diff
@@ -1186,8 +1183,13 @@ def _construct_exploration_suggestions(
     exp_id_to_exp = exp_fetchers.get_multiple_explorations_by_id(list(exp_ids))
     for suggestion in suggestions:
         exploration = exp_id_to_exp[suggestion.target_id]
-        content_html = exploration.get_content_html(
-            suggestion.change.state_name, suggestion.change.content_id)
+        content_html: Union[str, List[str]] = ''
```

Comment:
@sagangwee Can we set this to null instead? That is semantically more correct, otherwise it suggests that the content HTML is the empty string (which is a valid value).